app/api/PRMSCoverageTool.py

```diff
@@ -0,0 +1,106 @@
+__author__ = 'jerickson'
+
+import netCDF4
+import shutil
+import os
+
+### PRMS File representation
+class PRMSFile:
+    def __init__(self, basefile):
+        self.basefile = basefile
+        self.workingscenario = None
+
+    def BeginScenario(self, scenarioname):
+        '''
+        Starts a new scenario based on the original file under a new name
+
+        :param scenarioname: Name of scenario to make a new file; .nc will be appended
+        :return:
+        '''
+        if self.workingscenario is not None:
+            raise Exception("Working Scenario already open")
+            return
+        scenariofile = scenarioname + ".nc"
```

Comment:
Could this lead to a race condition if two users try to run a scenario at the same time? Would the OS take care of scheduling to make sure this is a safe operation? 

Also we need to add some kind of uniqueness or handling of repeat scenario names.